Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM registry.opensource.zalan.do/library/alpine-3.13:latest
+FROM container-registry.zalando.net/library/alpine-3.13:latest
```

Comment:
hmm, this image is now private which would prevent open-source users from building it :/

Reply:
Nothing I can do in the short-term I believe. I'll make it configurable.